data/entities.yaml

```
@@ -263,8 +263,15 @@
     attr: device
     char: 'C'
   description:
-  - An 8-bit counter.
+  - "Increment and reset, what more could you want?"
+  - |
+    A counter enables the command 'count : string -> cmd int',
+    which counts how many occurrences of an entity are currently
+    in the inventory.  Note also the 'has' command, which returns
+    a bool instead of an int and does not require any special
+    device.
```

I think this change would make the has/count situation clearer. Then again understanding text is subjective :slightly_smiling_face: 
```suggestion
    in the inventory.  This is an upgraded version of the 'has'
    command, which returns a bool instead of an int and does
    not require any special device.
```

---

I like it, thanks.